iati/core/tests/test_constants.py

```diff
@@ -13,3 +13,17 @@ def test_nsmap(self):
         """Check that the NSMAP constant is a dictionary with the correct key."""
         assert isinstance(iati.core.constants.NSMAP, dict)
         assert isinstance(iati.core.constants.NSMAP['xsd'], str)
+
+    def test_standard_versions_all_are_numbers(self):
+        """Check that each item in standard versions is a string that can be converted to a float."""
```

Comment:
Saying that it can be converted to a float is an implementation detail.

Suggest instead something about checking that each version number is a decimal number.